recipes/libevent/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package)
+project(test_package CXX)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
```

Comment:
Looking at `test_package.cpp`, it is actually c.
So you can rename it to `test_package.c`.

Reply:
good catch

Reply:
done